site/src/content/docs/forms/text-area.mdx

```diff
@@ -0,0 +1,186 @@
+</Callout>
+
+<Example code={`<div class="text-area">
+    <div class="text-area-container">
+      <label for="textAreaWithHelperText">Additional comment</label>
+      <textarea id="textAreaWithHelperText" class="text-area-field" aria-describedby="textAreaWithHelperTextHelpBlock"></textarea>
+    </div>
+    <p id="textAreaWithHelperTextHelpBlock" class="helper-text">
+      You have <strong>180</strong> characters left.
+    </p>
+  </div>`} />
+
+### Helper link
+
+If the helper text is not sufficient, it's possible to offer the user an additional help link. The helper link can also be displayed on its own without helper text.
+To display a helper link below inputs, use a standard small link [with `.link` and `.link-sm` classes]([[docsref:/components/links#sizes]]) as a sibling of a `.text-area-container`.
+
+<Callout type="warning">
+If a helper link is used in conjunction with a helper text, the `aria-describedby` attribute should refer to the helper text and the helper link should to be referenced by the `aria-labelledby` attribute.
+</Callout>
+
+<Example code={`<div class="text-area mb-medium">
+    <div class="text-area-container">
+      <label for="textAreaWithHelperTextLink">Additional comment</label>
+      <textarea id="textAreaWithHelperTextLink" aria-describedby="feedbackTextHelpBlock" aria-labelledby="feedbackTextHelpMore" class="text-area-field"></textarea>
+    </div>
+    <p id="feedbackTextHelpBlock" class="helper-text">
+      You have <strong>180</strong> characters left.
+    </p>
+    <a href="#" id="feedbackTextHelpMore" class="link link-sm">
+      More information
+    </a>
+  </div>
+  <div class="text-area">
+    <div class="text-area-container mt-lg">
+      <label for="textAreaHelperLink">Additional comment</label>
+      <textarea id="textAreaHelperLink" aria-describedby="feedbackTextHelpLink" class="text-area-field"></textarea>
+    </div>
+    <a href="#" id="feedbackTextHelpLink" class="link link-sm">
+      More information
+    </a>
+  </div>`} />
+
+## States
+
+### Disabled
+
+Add the `disabled` boolean attribute on an input to give it a grayed out appearance, remove pointer events, and prevent focusing.
+
+<Example code={`<div class="text-area mb-medium">
+    <div class="text-area-container">
+      <label for="exampleTextAreaDisabled">Additional comment</label>
+      <textarea class="text-area-field" id="exampleTextAreaDisabled" disabled>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer viverra nisi turpis, vel dignissim enim imperdiet eu. Ut quis mollis erat. Pellentesque id leo pellentesque urna volutpat placerat non nec sem. Integer mauris eros, congue nec magna sit amet, pulvinar laoreet diam. Nam tristique nisl ac nisi aliquet, et molestie turpis maximus.</textarea>
```

Comment:
I think that we can move the discriminant element early so it's easier to see what changed.

```suggestion
      <textarea disabled class="text-area-field" id="exampleTextAreaDisabled">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer viverra nisi turpis, vel dignissim enim imperdiet eu. Ut quis mollis erat. Pellentesque id leo pellentesque urna volutpat placerat non nec sem. Integer mauris eros, congue nec magna sit amet, pulvinar laoreet diam. Nam tristique nisl ac nisi aliquet, et molestie turpis maximus.</textarea>
```